### PR TITLE
javascript-loops-practice

### PR DESCRIPTION
**Pull request by Smith-Steve**
image

### CODE REVIEW

File: javascript-loops-practice/filter-out-strings.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/* exported filterOutStrings */
+function filterOutStrings(values) {
+  var newArray = [];
+  for (var i = 0; i < values.length; i++) {
+    if (typeof values[i] === 'string') {
+      // do nothing. Is there a better way to do this?
```
Comment (codingcodymiller) [inline javascript-loops-practice/filter-out-strings.js]: Yep! Your condition should be `typeof values[i] !== 'string'` so the `if` statement can be where you do the work you want.

File: javascript-loops-practice/filter-out-nulls.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/* exported filterOutNulls */
+function filterOutNulls(values) {
+  var newArray = [];
+  for (var i = 0; i < values.length; i++) {
+    if (values[i] != null) {
```
Comment (codingcodymiller) [inline javascript-loops-practice/filter-out-nulls.js]: You should use strictly not equal here `!==`

File: javascript-loops-practice/get-student-names.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+/* exported getStudentNames */
+function getStudentNames(students) {
+  var newArray = [];
+  for (var i = 0; i < students.length; i++) {
+    for (var student in students[i]) {
+      newArray.push(students[i][student]);
+    }
```
Comment (codingcodymiller) [inline javascript-loops-practice/get-student-names.js]: You don't need this nested loop. You only need the value of the `name` property, and you know the specific name of that property, so you can just use dot notation to access that property

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Nice work so far, dude!! A couple notes for you:

**Comment by Smith-Steve** [comment]
image

**Review by codingcodymiller — APPROVED** [review]
Nice job!!

**Pull request merged**